site/components/ReviewConsequenceTable.tsx

```diff
@@ -0,0 +1,252 @@
+import Link from "next/link";
+import { ReactElement, ReactNode } from "react";
+import Table from "./form/Table";
+import {
+    CONSEQUENCE_TYPES,
+    DISRUPTION_DETAIL_PAGE_PATH,
+    REVIEW_DISRUPTION_PAGE_PATH,
+    TYPE_OF_CONSEQUENCE_PAGE_PATH,
+    VEHICLE_MODES,
+} from "../constants";
+import { Consequence } from "../schemas/consequence.schema";
+import { Disruption } from "../schemas/disruption.schema";
+import { getDisplayByValue, splitCamelCaseToString } from "../utils";
+
+const getConsequenceUrl = (type: Consequence["consequenceType"]) => {
+    switch (type) {
+        case "networkWide":
+            return "/create-consequence-network";
```

Comment:
Can we use the constants for these links?

Reply:
done